src/usePatchRef/usePatchRef.ts

```diff
@@ -0,0 +1,15 @@
+import { Ref, computed } from 'vue'
+
+export function usePatchRef<T extends Record<string, unknown>, P extends keyof T>(source: Ref<T>, key: P): Ref<T[P]> {
```

Comment:
Could you add a docstring to this? Might make it easier to understand when someone is reading a component